README.md

```diff
@@ -1 +1,158 @@
-# maths-magician
```

Comment:
![image](https://github.com/Developer-Kingz/maths-magician/assets/93039359/03af01c4-f2d9-4924-9d11-82154e8c0547)

- [ ] It would be nice if you remove all the boilerplate that got created by the CRA (texts, images, and styles etc) as suggested by the previous reviewer 👍